dotnet/src/Connectors/Connectors.HuggingFace/Core/HuggingFacePromptExecutionSettings.cs

```diff
@@ -0,0 +1,223 @@
+﻿// Copyright (c) Microsoft. All rights reserved.
+
+using System;
+using System.Collections.Generic;
+using System.Text.Json;
+using System.Text.Json.Serialization;
+using Microsoft.SemanticKernel.Text;
+
+namespace Microsoft.SemanticKernel.Connectors.HuggingFace.Core;
```

Comment:
Can we remove the ".Core" in the namespace?

Reply:
Moved, good catch.